### DISCUSSION TITLE
format not working

### DISCUSSION THREAD

**Discussion by Menna-elgallad** [nuxt/image#941] [Q&A]
using format does not make effect i though maybe because i disable the ssr but when enabling it also doesn't work ? 
`nuxt-img(:src="src" v-if="src" loading="lazy"  @load="addclass" format="webp" )`

**Comment by Sadarex** [comment]
`format` is only available for `<nuxt-picture>`
https://image.nuxtjs.org/configuration#format

**Reply by Menna-elgallad** [reply]
it's also not working

**Reply by Sadarex** [reply]
Please check the files with a text-editor. Maybe only the file extension is wrong.

Look here for this issue:
https://github.com/nuxt/image/issues/933#issuecomment-1665158059
